appveyor.yml

```diff
@@ -1,6 +1,9 @@
 # AppVeyor.com is a Continuous Integration service to build and run tests under Windows
 # see https://github.com/ogrisel/python-appveyor-demo/blob/master/appveyor.yml
 
+build_cloud: PC1577-AS
+image: Windows
```

Comment:
looks like appveyor didnt even start a build with these changes? @ThomasLecocq 
Anyway the original PR looks good, no?

Reply:
Ah, maybe just glitched out, starting to build now after manually retriggering appveyor

Reply:
:-) yep